font-lohit-beng-extra.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: font-lohit-beng-extra
+  version: 3.3.1-2
+  epoch: 0
+  description: This package provides TrueType fonts for Bengali language which is primarily spoken in the Indian state of West Bengal, Assam, Tripura and in the country Bangladesh.
+  copyright:
+    - license: GPL-2.0-or-later AND GPL-2.0-or-later WITH Font-exception-2.0 AND GPL-3.0-or-later WITH Font-exception-2.0
```

Comment:
Based on the [SPDX documentation](https://spdx.org/licenses/GPL-2.0-with-font-exception.html) I believe this is the right way to specify the licenses [in this file](https://salsa.debian.org/fonts-team/fonts-beng-extra/-/blob/debian/3.3.1-2/debian/copyright).